theano/gpuarray/sort.py

```diff
@@ -120,7 +135,8 @@ def build_kernel(fname, kname, subs):
             ) as f:
                 kernel_src = f.read()
             ker = Kernel(
-                code=Template(common_src + kernel_src).substitute(**subs),
+                code=("#include <cluda.h>\n" +
```

Comment:
could be cleaner putting this inside `theano/gpuarray/c_code/topk_common.cuh`

Reply:
I did it this way to make sure it's at the top since it wasn't clear which file was read first.  I have no objection to moving it.